Models/UserData.cs

```diff
@@ -0,0 +1,30 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Models
+{
+    //This class will extend all future data regarding users, apart from the user-object itself.
+    public class UserData
+    {
+        public User User { get; set; }
+        
```

Comment:
I would say that all the projects created by the user are part of his/her userdata

Reply:
These are present in the user object itself and will thus be retrieved or deleted aswell. The userdata might contain stuff about userpreferences etc in the future